Nidhi.md

```diff
@@ -13,7 +12,28 @@
 - SRS appendix added.
 - SDLC contributed two sections
 
+# 29 Feb
+
+### Work
+[x] Survey v2 helped design edit and review.
```

Comment:
@berzentine sorry to bother you again.
but the file is not coming out right. You can see for yourself in your own repo.

I would suggest changing
`[x]` to `- [x]`